tools/analysis/manticore/log/execution_errors.md

```diff
@@ -0,0 +1,75 @@
+### PIP instalation:
```

Comment:
Q: what's the goal of this README? Is it just to document errors that developers using manticore may hit and how to navigate around it?
If so you should mention this so people know how to use it.

If these are repeatable and consistent maybe instead open tickets on the manticore repo with the details and reference those tickets here. That way updates and potential future improvements are captured